components/Filter.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { Button } from "@mui/material";
+
+const FilterContainer = styled.div`
+  display: flex;
+  gap: 10px;
+  padding: 10px;
+`;
+
+const expertises = [
+  "All",
+  "AI&Data",
+  "Engineering",
+  "Business",
+  "Product",
+  "Marketing",
+  "Operations",
+  "Investments",
+];
+
+export default function Filter({ onExpertiseSelect }) {
+  const [selectedExpertiseTag, setSelectedExpertiseTag] = useState("All");
+
+  const handleExpertiseClick = (expertise) => {
+    setSelectedExpertiseTag(expertise);
+    onExpertiseSelect(expertise);
+  };
+
+  return (
+    <FilterContainer>
+      {expertises.map((expertise) => (
+        <Button
+          key={expertise}
+          onClick={() => handleExpertiseClick(expertise)}
+          variant={
+            selectedExpertiseTag === expertise ? "contained" : "outlined"
+          }
```

Comment:
nice varaint naming

Reply:
The main problem is, that Spiced and Neuefische have a big gap in the capstone philosophy, and your cohort just has to bridge it. We have a lot of misunderstandings between coaches. As I wrote to you in Slack you can continue with MUI.